### PR TITLE
Revert Docker Install 20.10.8, 20.10.9, 20.10.10 and 20.10.11

### PR DESCRIPTION
**Pull request by rayandas**
This PR revert the recent changes where docker versions 20.10.8, 20.10.9, 20.10.10 and 20.10.11 got added. 
Also this added the symlink `20.10.sh` with `20.10.7.sh`

### CODE REVIEW

**Comment by matdehaast** [comment]
@rayandas what was the reasoning to revert the above? 

We are using the above script on Ubuntu 21.10 which requires docker > 20.10.10. This change has appeared to already be pushed to the actual symlinked `20.10.sh` file. This broke our CI environment that was relying on it.

I just want to understand the reasoning to revert and also remove all the scripts.

Thanks

**Review by kinarashah — APPROVED** [review]

**Comment by kinarashah** [comment]
https://github.com/rancher/install-docker/pull/83/

**Review by superseb — APPROVED** [review]

**Comment by rayandas** [comment]
Seems like I don't have merge rights. @superseb could you please merge this? I will raise another PR after that to fix the centos8 CI failures. Similarly what rke2 team did.

**Pull request merged**
